hmy/api_backend.go

```diff
@@ -207,6 +207,12 @@ func (b *APIBackend) GetBalance(address common.Address) (*hexutil.Big, error) {
 	return (*hexutil.Big)(balance), err
 }
 
+// GetTransactionsHistory returns list of transactions hashes of address.
+func (b *APIBackend) GetTransactionsHistory(address string) ([]common.Hash, error) {
```

Comment:
Why do we only return hashes? Should we add pagination? The number of txns can be huge.

Reply:
I guess we can show 100 of txs by default, but shouldn't it be only on frontend side?